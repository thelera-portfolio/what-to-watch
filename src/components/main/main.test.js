import React from "react";
import configureStore from "redux-mock-store";
import {Provider} from "react-redux";
import renderer from "react-test-renderer";
import Main from "./main.jsx";

const mockStore = configureStore([]);

const filmData = {
  image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  genres: [`Horror`],
  title: `No Country for Old Men`,
  year: 2005,
};

const films = [
  {
    title: `Bekket`,
    image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    title: `The Doom Generation`,
    image: `img/bohemian-rhapsody.jpg`,
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    title: `Patrool`,
    image: `img/macbeth.jpg`,
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
];

it(`Main should render movie card and films catalog`, () => {
  const store = mockStore({
    allFilms: films,
    genre: `All genres`,
  });

  const tree = renderer.create(
      <Provider store={store}>
        <Main
          filmsList={films}
          promoFilm={filmData}
        />
      </Provider>, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
