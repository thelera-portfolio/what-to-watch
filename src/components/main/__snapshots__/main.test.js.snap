// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render movie card and films catalog 1`] = `
<div>
  <section
    className="movie-card"
  >
    <div
      className="movie-card__bg"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      />
    </div>
    <h1
      className="visually-hidden"
    >
      WTW
    </h1>
    <header
      className="page-header movie-card__head"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="user-block"
      >
        <a
          className="user-block__link"
          href="/mylist"
          onClick={[Function]}
        >
          <div
            className="user-block__avatar"
          >
            <img
              alt="User avatar"
              height="63"
              src="url"
              width="63"
            />
          </div>
        </a>
      </div>
    </header>
    <div
      className="movie-card__wrap"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster"
        >
          <img
            alt="Fantastic Beasts: The Crimes of Grindelwald"
            height="327"
            src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Fantastic Beasts: The Crimes of Grindelwald
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Fantasy
            </span>
            <span
              className="movie-card__year"
            >
              2015
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <a
              className="btn btn--play movie-card__button"
              href="/player/345712414"
              onClick={[Function]}
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </a>
            <button
              className="btn btn--list movie-card__button"
              onClick={[Function]}
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    className="page-content"
  >
    <section
      className="catalog"
    >
      <h2
        className="catalog__title visually-hidden"
      >
        Catalog
      </h2>
      <ul
        className="catalog__genres-list"
      >
        <li
          className="catalog__genres-item catalog__genres-item--active"
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            All genres
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Adventure
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Comedy
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Crime
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Documentary
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Drama
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Fantasy
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Horror
          </a>
        </li>
        <li
          className="catalog__genres-item "
        >
          <a
            className="catalog__genres-link"
            onClick={[Function]}
          >
            Kid's and Family
          </a>
        </li>
      </ul>
      <div
        className="catalog__movies-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
        >
          <a
            className="small-movie-card__image"
            href="/films/1"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="visually-hidden"
            >
              <svg
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <symbol
                  id="add"
                  viewBox="0 0 19 20"
                >
                  <title>
                    +
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Page-1"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      id="+"
                      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
                    />
                  </g>
                </symbol>
                <symbol
                  id="full-screen"
                  viewBox="0 0 27 27"
                >
                  <path
                    clipRule="evenodd"
                    d="M23.8571 0H16V3.14286H23.8571V11H27V3.14286V0H23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M27 23.8571V16H23.8571V23.8571H16V27H23.8571H27L27 23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 3.14286L0 11H3.14286L3.14286 3.14286L11 3.14286V0H3.14286H0L0 3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M3.14286 27H11V23.8571H3.14286L3.14286 16H0L0 23.8571V27H3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="in-list"
                  viewBox="0 0 18 14"
                >
                  <path
                    clipRule="evenodd"
                    d="M2.40513 5.35353L6.1818 8.90902L15.5807 0L18 2.80485L6.18935 14L0 8.17346L2.40513 5.35353Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="pause"
                  viewBox="0 0 14 21"
                >
                  <title>
                    Artboard
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Artboard"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
                    />
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
                    />
                  </g>
                </symbol>
                <symbol
                  id="play-s"
                  viewBox="0 0 19 19"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 0L19 9.5L0 19V0Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
              </svg>
            </div>
            <video
              className=""
            >
              Sorry, your browser doesn't support embedded videos.
            </video>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/1"
              onClick={[Function]}
            >
              Fantastic Beasts: The Crimes of Grindelwald
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
        >
          <a
            className="small-movie-card__image"
            href="/films/2"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="visually-hidden"
            >
              <svg
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <symbol
                  id="add"
                  viewBox="0 0 19 20"
                >
                  <title>
                    +
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Page-1"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      id="+"
                      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
                    />
                  </g>
                </symbol>
                <symbol
                  id="full-screen"
                  viewBox="0 0 27 27"
                >
                  <path
                    clipRule="evenodd"
                    d="M23.8571 0H16V3.14286H23.8571V11H27V3.14286V0H23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M27 23.8571V16H23.8571V23.8571H16V27H23.8571H27L27 23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 3.14286L0 11H3.14286L3.14286 3.14286L11 3.14286V0H3.14286H0L0 3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M3.14286 27H11V23.8571H3.14286L3.14286 16H0L0 23.8571V27H3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="in-list"
                  viewBox="0 0 18 14"
                >
                  <path
                    clipRule="evenodd"
                    d="M2.40513 5.35353L6.1818 8.90902L15.5807 0L18 2.80485L6.18935 14L0 8.17346L2.40513 5.35353Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="pause"
                  viewBox="0 0 14 21"
                >
                  <title>
                    Artboard
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Artboard"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
                    />
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
                    />
                  </g>
                </symbol>
                <symbol
                  id="play-s"
                  viewBox="0 0 19 19"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 0L19 9.5L0 19V0Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
              </svg>
            </div>
            <video
              className=""
            >
              Sorry, your browser doesn't support embedded videos.
            </video>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/2"
              onClick={[Function]}
            >
              Bohemian Rhapsody
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
        >
          <a
            className="small-movie-card__image"
            href="/films/3"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="visually-hidden"
            >
              <svg
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <symbol
                  id="add"
                  viewBox="0 0 19 20"
                >
                  <title>
                    +
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Page-1"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      id="+"
                      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
                    />
                  </g>
                </symbol>
                <symbol
                  id="full-screen"
                  viewBox="0 0 27 27"
                >
                  <path
                    clipRule="evenodd"
                    d="M23.8571 0H16V3.14286H23.8571V11H27V3.14286V0H23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M27 23.8571V16H23.8571V23.8571H16V27H23.8571H27L27 23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 3.14286L0 11H3.14286L3.14286 3.14286L11 3.14286V0H3.14286H0L0 3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M3.14286 27H11V23.8571H3.14286L3.14286 16H0L0 23.8571V27H3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="in-list"
                  viewBox="0 0 18 14"
                >
                  <path
                    clipRule="evenodd"
                    d="M2.40513 5.35353L6.1818 8.90902L15.5807 0L18 2.80485L6.18935 14L0 8.17346L2.40513 5.35353Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="pause"
                  viewBox="0 0 14 21"
                >
                  <title>
                    Artboard
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Artboard"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
                    />
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
                    />
                  </g>
                </symbol>
                <symbol
                  id="play-s"
                  viewBox="0 0 19 19"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 0L19 9.5L0 19V0Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
              </svg>
            </div>
            <video
              className=""
            >
              Sorry, your browser doesn't support embedded videos.
            </video>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/3"
              onClick={[Function]}
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
        >
          <a
            className="small-movie-card__image"
            href="/films/4"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="visually-hidden"
            >
              <svg
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <symbol
                  id="add"
                  viewBox="0 0 19 20"
                >
                  <title>
                    +
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Page-1"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      id="+"
                      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
                    />
                  </g>
                </symbol>
                <symbol
                  id="full-screen"
                  viewBox="0 0 27 27"
                >
                  <path
                    clipRule="evenodd"
                    d="M23.8571 0H16V3.14286H23.8571V11H27V3.14286V0H23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M27 23.8571V16H23.8571V23.8571H16V27H23.8571H27L27 23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 3.14286L0 11H3.14286L3.14286 3.14286L11 3.14286V0H3.14286H0L0 3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M3.14286 27H11V23.8571H3.14286L3.14286 16H0L0 23.8571V27H3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="in-list"
                  viewBox="0 0 18 14"
                >
                  <path
                    clipRule="evenodd"
                    d="M2.40513 5.35353L6.1818 8.90902L15.5807 0L18 2.80485L6.18935 14L0 8.17346L2.40513 5.35353Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="pause"
                  viewBox="0 0 14 21"
                >
                  <title>
                    Artboard
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Artboard"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
                    />
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
                    />
                  </g>
                </symbol>
                <symbol
                  id="play-s"
                  viewBox="0 0 19 19"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 0L19 9.5L0 19V0Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
              </svg>
            </div>
            <video
              className=""
            >
              Sorry, your browser doesn't support embedded videos.
            </video>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/4"
              onClick={[Function]}
            >
              Aviator
            </a>
          </h3>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
        >
          <a
            className="small-movie-card__image"
            href="/films/5"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="visually-hidden"
            >
              <svg
                xmlns="http://www.w3.org/2000/svg"
                xmlnsXlink="http://www.w3.org/1999/xlink"
              >
                <symbol
                  id="add"
                  viewBox="0 0 19 20"
                >
                  <title>
                    +
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Page-1"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      id="+"
                      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
                    />
                  </g>
                </symbol>
                <symbol
                  id="full-screen"
                  viewBox="0 0 27 27"
                >
                  <path
                    clipRule="evenodd"
                    d="M23.8571 0H16V3.14286H23.8571V11H27V3.14286V0H23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M27 23.8571V16H23.8571V23.8571H16V27H23.8571H27L27 23.8571Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 3.14286L0 11H3.14286L3.14286 3.14286L11 3.14286V0H3.14286H0L0 3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                  <path
                    clipRule="evenodd"
                    d="M3.14286 27H11V23.8571H3.14286L3.14286 16H0L0 23.8571V27H3.14286Z"
                    fill="#FFF9D9"
                    fillOpacity="0.7"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="in-list"
                  viewBox="0 0 18 14"
                >
                  <path
                    clipRule="evenodd"
                    d="M2.40513 5.35353L6.1818 8.90902L15.5807 0L18 2.80485L6.18935 14L0 8.17346L2.40513 5.35353Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
                <symbol
                  id="pause"
                  viewBox="0 0 14 21"
                >
                  <title>
                    Artboard
                  </title>
                  <desc>
                    Created with Sketch.
                  </desc>
                  <g
                    fill="none"
                    fillRule="evenodd"
                    id="Artboard"
                    stroke="none"
                    strokeWidth="1"
                  >
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
                    />
                    <polygon
                      fill="#EEE5B5"
                      fillRule="nonzero"
                      id="Line"
                      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
                    />
                  </g>
                </symbol>
                <symbol
                  id="play-s"
                  viewBox="0 0 19 19"
                >
                  <path
                    clipRule="evenodd"
                    d="M0 0L19 9.5L0 19V0Z"
                    fill="#EEE5B5"
                    fillRule="evenodd"
                  />
                </symbol>
              </svg>
            </div>
            <video
              className=""
            >
              Sorry, your browser doesn't support embedded videos.
            </video>
          </a>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/5"
              onClick={[Function]}
            >
              We Need To Talk About Kevin
            </a>
          </h3>
        </article>
      </div>
      <div
        className="catalog__more"
      />
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
